lib/sanbase/discourse/insight.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Sanbase.Discourse.Insight do
+  require Sanbase.Utils.Config, as: Config
+
+  alias Sanbase.Insight.Post
+  alias Sanbase.Discourse
+
+  def create_discourse_topic(%Post{id: id, title: title, inserted_at: inserted_at}) do
+    link = posts_url(id)
+
+    text = ~s"""
+      This topic hosts the discussion about [#{link}](#{link})
+    """
+
+    title = "##{id} | #{title} | #{DateTime.to_naive(inserted_at) |> to_string}"
```

Comment:
```suggestion
    title = "##{id} | #{title} | #{inserted_at |> DateTime.truncate(:second) |> DateTime.to_naive()}"
```

`to_string` is not needed as DateTime implements the String.Chars protocol. `truncate` is for better title as we don't care for microseconds here